polynote-frontend/style/styles.less

```diff
@@ -304,12 +310,13 @@ body {
       bottom: 0;
       width: @cell-margin-width;
       background: @ui-background;
-      border: 1px solid @ui-border;
+      border-top-left-radius: 10px;
+      box-shadow: -1px 1px 2px rgba(0, 0, 0, 0.25);
       padding: 4px;
       box-sizing: border-box;
 
       button {
-        display: block;
+        display: none;
```

Comment:
Doesn't this hide all the run buttons? Why?

Reply:
Oh, I see – you only show the run button for the active cell. Would users like that? I guess jupyter doesn't have any run buttons at all... 🤷‍♂️

Reply:
Would that mean that to run a cell I'd need two clicks - one to select and then another to run? I kind of like the ability to click once to run a cell. 

On the other hand, if our intention is to nudge people to prefer not to run cells out of order perhaps we shouldn't necessarily be making it easy for them to do so 😉

Reply:
Well, I think very few people run a cell by clicking the button. It's almost always "shift-enter" to run. As Jeremy mentions Jupyter doesn't even have a run button !! So having it there for active cells is already 100% more explicit.

Having them for ever cell looks cluttered IMO. But I'm probably a web2.0 style hipster  :)